terms.html

```diff
@@ -71,7 +72,7 @@
 </ul>
 <p>in any applicable law (which may depend on where you are located and any jurisdiction to which your activities are directed).</p>
 <h2>Use of cookies</h2>
```

Comment:
As a matter of good practice (and I think, law, though the ICO has been very slack on enforcement) you should document cookies somewhere. I suggest having a cookie page with a table of what cookies are set, what they are set by and what they are for. A little extra work I am afraid.

Strictly, you should be warning all visitors to your site about cookies and getting their explicit consent to set them.